peekingduck/pipeline/nodes/model/mtcnnv1/mtcnn_model.py

```diff
@@ -46,15 +46,19 @@ def __init__(self, config: Dict[str, Any]) -> None:
         if not 0 <= config["mtcnn_score"] <= 1:
             raise ValueError("mtcnn_score must be between 0 and 1")
 
+        weights_dir, model_dir = finder.find_paths(
+            config["root"], config["weights_parent_dir"], config["weights"]
+        )
+
         # check for mtcnn weights, if none then download into weights folder
-        if not checker.has_weights(config["root"], config["weights_dir"]):
+        if not checker.has_weights(weights_dir, model_dir):
             self.logger.info(
                 "---no mtcnn weights detected. proceeding to download...---"
             )
-            downloader.download_weights(config["root"], config["blob_file"])
-            self.logger.info("---mtcnn weights download complete.---")
+            downloader.download_weights(weights_dir, config["weights"]["blob_file"])
+            self.logger.info(f"---mtcnn weights downloaded to {weights_dir}.---")
```

Comment:
This may be beyond the scope of this PR, but since the logger has the module/file's name (mtcnn_model), maybe the logging messages can be simplified to just "No weights detected. Proceeding to download ..." instead of repeating the model name?

Reply:
Model name has been removed for all 7 models and tests updated.